beluga/test/beluga/algorithm/test_distance_map.cpp

```diff
@@ -12,62 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <vector>
+
 #include <gmock/gmock.h>
 
 #include <beluga/algorithm/distance_map.hpp>
 
+#include <Eigen/Core>
+
 namespace {
 
-auto array_distance(std::size_t first, std::size_t second) {
-  return std::abs(static_cast<intmax_t>(first) - static_cast<intmax_t>(second));
+using CellType = Eigen::Vector2i;
+
+auto array_distance(const CellType& first, const CellType& second) {
```

Comment:
@glpuga nit: this is no longer an "array" distance but a regular distance in euclidean space.

Reply:
Fixed.